terraform/README-terraform.md

```diff
@@ -0,0 +1,245 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- Azure Spring Cloud
+description: "Deploy Spring microservices using Azure Spring Cloud and MySQL"
+urlFragment: "spring-petclinic-microservices"
+---
+# Deploy Spring Microservices using Azure Spring Cloud and MySQL 
+
+Azure Spring Cloud enables you to easily run a Spring Boot based microservices application on Azure.
+
+This quickstart shows you how to deploy an existing Java Spring Cloud application to Azure. When you're finished, you can continue to manage the application via the Azure CLI or switch to using the Azure portal.
+
+## What will you experience
+You will:
+- Build existing Spring microservices applications
+- Provision an Azure Spring Cloud service instance
```

Comment:
Should we call out that they will be using Terraform?

Reply:
Fixed.